doc/protocol.md

```diff
@@ -794,8 +794,7 @@ following options can be specified:
  * "err": If "spawn" is set, and "err" is set to "out", then stderr
    is included in the payload data. If "err" is set to "ignore" then, the
    stderr output will be discarded. If "err" is set to "message" then it will
-   be included in the close message. If "pty" is set then stderr is always
```

Comment:
This statement is true but slightly ambiguous, but should definitely not be removed.  In the "pty" mode stderr is always basically set to "out" — ie: it gets delivered in the main channel payload.

Note in particular: this definitely doesn't say that "pty" is a valid value for "err".